main.py

```diff
@@ -1 +1,23 @@
 #Mad Libs Generator Project
+//Loop back to this point once code finishes
+loop = 1
+while (loop < 9):
```

Comment:
Is there a reason that the while loop is executing multiple times? This code is prompting the user for all of the inputs and also prints out the paragraph for the mad lib 8 times.

Reply:
Instead of a while loop, let's create a main function called something like "play_mad_libs_six()" that contains two helper functions, one to get user input, and the other to print the completed mad lib. We can erase the while loop if we do this.